### PR TITLE
Fix linting issues

### PR DESCRIPTION
**Pull request by krishanthisera**
### Fix linting issues

------

**Description of the proposed changes**  

* Fix linting errors

**Screenshots (if applicable)**  

* N/A

**Other solutions considered (if any)**  

* N/A

**Notes to PR author**

⚠️  Please make sure the changes are adhere to the guidelines mention [here](https://aligent.atlassian.net/wiki/spaces/AL/pages/2758836297/Development+checklist+FE+BE+INT)

**Notes to reviewers**  
* Next we might need to add checks to the GitHub action

### CODE REVIEW

File: packages/lambda-at-edge-handlers/lib/prerender-check.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import "source-map-support/register";
 import { CloudFrontRequest, CloudFrontRequestEvent } from "aws-lambda";
 
 const IS_BOT =
-  /googlebot|Google-InspectionTool|chrome-lighthouse|lighthouse|adsbot\-google|Feedfetcher\-Google|bingbot|yandex|baiduspider|Facebot|facebookexternalhit|twitterbot|rogerbot|linkedinbot|embedly|quora link preview|showyoubot|outbrain|pinterest|slackbot|vkShare|W3C_Validator/i;
+  /googlebot|Google-InspectionTool|chrome-lighthouse|lighthouse|adsbot-google|Feedfetcher-Google|bingbot|yandex|baiduspider|Facebot|facebookexternalhit|twitterbot|rogerbot|linkedinbot|embedly|quora link preview|showyoubot|outbrain|pinterest|slackbot|vkShare|W3C_Validator/i;
```
Comment (TheOrangePuff) [inline packages/lambda-at-edge-handlers/lib/prerender-check.ts:5]: @krishanthisera just confirming this is an expected change?

File: packages/prerender-proxy/lib/handlers/prerender-check.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ import "source-map-support/register";
 import { CloudFrontRequest, CloudFrontRequestEvent } from "aws-lambda";
 
 const IS_BOT =
-  /googlebot|Google-InspectionTool|chrome-lighthouse|lighthouse|adsbot\-google|Feedfetcher\-Google|bingbot|yandex|baiduspider|Facebot|facebookexternalhit|twitterbot|rogerbot|linkedinbot|embedly|quora link preview|showyoubot|outbrain|pinterest|slackbot|vkShare|W3C_Validator/i;
```
Comment (TheOrangePuff) [inline packages/prerender-proxy/lib/handlers/prerender-check.ts:5]: @krishanthisera same with this. Also why is this code duplicated?

File: packages/static-hosting/lib/static-hosting.ts [hunk 3]
```diff
@@ -495,7 +495,7 @@ export class StaticHosting extends Construct {
     // If the remap is to a different path, create a Lambda@Edge function to handle this
     if (from !== to) {
       // Remove special characters from path
-      const id = from.replace(/[&\/\\#,+()$~%'":*?<>{}]/g, "-");
+      const id = from.replace(/[&/\\#,+()$~%'":*?<>{}]/g, "-");
```
Comment (TheOrangePuff) [inline packages/static-hosting/lib/static-hosting.ts:498]: @krishanthisera is this expected?

**Review by TheOrangePuff — CHANGES_REQUESTED** [review]
@krishanthisera a couple questions around the regex linting

**Review by crispy101 — APPROVED** [review]
LGTM

File: packages/lambda-at-edge-handlers/lib/prerender-check.ts (hunk 1 above)
Comment (krishanthisera) [reply packages/lambda-at-edge-handlers/lib/prerender-check.ts:5]: Yeah, we have escape character, which is not necessary `|adsbot\-google`

File: packages/static-hosting/lib/static-hosting.ts (hunk 3 above)
Comment (krishanthisera) [reply packages/static-hosting/lib/static-hosting.ts:498]: This is expected, again unnecessary escape character

File: packages/prerender-proxy/lib/handlers/prerender-check.ts (hunk 2 above)
Comment (krishanthisera) [reply packages/prerender-proxy/lib/handlers/prerender-check.ts:5]: Again unnecessary escape character. These are two different packages, one is to be used as a library when you want to use the Lambda functions with Middleware.

**Review by TheOrangePuff — APPROVED** [review]

**Pull request merged**